generate-diff.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+jdiff_version='1.1.0'
+set -x
```

Comment:
This can be removed, but might we worth keeping to have logs of all commands that are being ran.